_posts/2025-06-05-two-arg-open.md

```diff
@@ -0,0 +1,40 @@
+---
+layout: post
+published: true
+toc: true
+title: "Are you still using the 2-argument open?"
+date: 2025-06-05 12:00:00 +0100
+tags: authors guides cpan modules security
+author: Robert Rothenberg
+excerpt: "The 2-argument open function is insecure"
+---
+
+The 2-argument [open](https://metacpan.org/pod/perlfunc#open) function is insecure, because the filename can include the mode.
+If it is not properly validated, then files can be modified, truncated or a pipe to run an external command:
```

Comment:
The pipe portion reads a little awkwardly.  

or in the case of a pipe character, coerced to run an external command.

Reply:
Not sure coerced is right either

Reply:
Just simply " in the case of a pipe character, run an external command".